### PR TITLE
Add Sierra leone flag

### PR DESCRIPTION
**Pull request by TS96**
I added Sierra Leone's flag, if they're any issues, I'd be happy to fix them.

### CODE REVIEW

File: Flags/SL.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Sierra Leone | Flags</title>
+    <style>
+      #sl{
```
Comment (ManrajGrover) [inline Flags/SL.html]: Please change the id to full name of country. :) Make the same change in file name and image name.

File: README.md [hunk 2]
```diff
@@ -44,6 +44,7 @@ One `<div>`. Many possibilities.
 | [Japan](https://github.com/ManrajGrover/SingleDivProject/blob/master/Flags/Japan.html) | [ManrajGrover](https://github.com/ManrajGrover) | ![Japan](https://github.com/ManrajGrover/SingleDivProject/blob/master/Assets/Japan.png) |
 | [Russia](https://github.com/ManrajGrover/SingleDivProject/blob/master/Flags/Russia.html) | [ad222kr](https://github.com/ad222kr) | ![Russia](https://github.com/ManrajGrover/SingleDivProject/blob/master/Assets/Russia.png) |
 | [South Africa](https://github.com/ManrajGrover/SingleDivProject/blob/master/Flags/South%20Africa.html) | [DHernandex](https://github.com/dhernandex) | ![South Africa](https://github.com/ManrajGrover/SingleDivProject/blob/master/Assets/South%20Africa.png)
+| [Sierra Leone](https://github.com/ManrajGrover/SingleDivProject/blob/master/Flags/SL.html) | [ts96](https://github.com/ts96) | ![Sierra Leone](https://github.com/ManrajGrover/SingleDivProject/blob/master/Assets/SL.png) |
```
Comment (ManrajGrover) [inline README.md]: Please make the same changes here. :)

**Comment by ManrajGrover** [comment]
@TS96 This is great! Thank you for contributing. I have mentioned some changes. Please fix them so we can merge it. :smile:

**Comment by TS96** [comment]
@ManrajGrover Hi, I fixed all the issues and if there others, I'd be happy to fix them as well. :)

**Comment by ManrajGrover** [comment]
@TS96 Awesome. Thank you for contributing. :smile: Merging this.

**Pull request merged**